src/functions/newsletter-subscription/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "newsletter-subscription",
+    "version": "0.1.0",
+    "description": "MuShop Newsletter Subscription",
+    "main": "func.js",
+    "author": "",
+    "license": "Apache-2.0",
```

Comment:
We may need to change to UPL

Reply:
This was the default - who do we need to check with to change it?

Reply:
Looks like the api/storefront `package.json` doesn't have the license defined, so we probably fallback to whatever the repo has. Will remove it.